src/documenti/esterni/ManualeUtente/contenuti/istruzioniUso.tex

```diff
@@ -1 +1,33 @@
 \chapter{Istruzioni all'uso}
+Il seguente capitolo fornirà tutte le spiegazioni per il corretto utilizzo del prodotto.
+
+\section{Home}
+Questa è la prima schermata disponibile del prodotto. Qui è possibile caricare i dati sotto forma di dataset o di sessione salvata in precedenza.
```

Comment:
"schermata iniziale quando viene aperta l'applicazione" . direi che in questo documento non ha molto senso usare la parola prodotto